src/commands/txCommands/util/index.js

```diff
@@ -381,6 +369,48 @@ const resourcesComplete$ = resourceCompletion$
 	.filter(item => Object.keys(item[1]).length === 0)
 	.map(([resource]) => resource);
 
+// Helper to grab all local trns to be merged to a tx resource (e.g. default English messages)
+const updateAllMessages$ = (resource, project) =>
+	localTrnsMerged$
+		.flatMap(poContent =>
+			updateResource$(
+				resource,
+				poContent,
+				project,
+			).do(
+				() => console.log(`update ${project} - ${resource} success`),
+				() => console.log(`update ${project} - ${resource}  FAIL!`))
```

Comment:
there's an `error` argument passed to this function - you should print it out just in case it has useful debug info